_posts/2018-11-23-public_to_github-page.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: How to public your posts to github-page?
+date: 2018-11-23
+categories: [linux, tutorials, kubernetes]
+tags: [Linux, Tutorials, Kubernetes]
+---
+
+1. **Fork** the repo https://github.com/vietkubers/vietkubers.github.io to your Github account
+2. **Clone** the above forked repo to your local directory
+```
+$ git clone https://github.com/$USER/vietkubers.github.io.git
+```
+3. **Create** a [markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) file in folder [_post](https://github.com/vietkubers/vietkubers.github.io/tree/master/_posts) and name it as `yyyy-mm-dd-ten-bai-viet.md`
+```
+$ cd vietkubers.github.io.git/_post
+$ touch yyyy-mm-dd-ten-bai-viet.md
+```
+4. **Modify** the created post with declaration section as:
+```
+---
+layout: post
+title: Deploying multiple nodes with Kubeadm
+date: 2018-11-16
+categories: [linux, tutorials, kubernetes]
+tags: [Linux, Tutorials, Kubernetes]
+---
+```
+Please refer to [this example post](https://raw.githubusercontent.com/vietkubers/vietkubers.github.io/master/_posts/2018-11-21-deploying-multiplenodes-with-kubeadm.md) for more details.  
+
+5.  Commit, push and create your Pull Request
+
+Refer to this article: https://vietkubers.github.io/linux/tutorials/kubernetes/2018/11/16/github-workflow.html for the step-by-step process.  
```

Comment:
I think we should use reference instead of using direct URL here. For example:

Refer to this article [1]
## References
[1] https://vietkubers.github.io/linux/tutorials/kubernetes/2018/11/16/github-workflow.html